src/_00_Sorting_Algorithms/_01_SwappingDigits.java

```diff
@@ -7,37 +7,66 @@
 class _01_SwappingDigits {
 
 	@Test
-	void test() {
-		int[] x = {5, 6};
-		int[] y = {6, 5};
+	void testInt() {
+		int[] x = { 5, 6 };
+		int[] y = { 6, 5 };
 		swapArrayOfTwo(x);
 		assertArrayEquals(y, x);
-		
-		int[] z = {7, 3, 5, 9, 2};
-		int[] q = {2, 3, 5, 7, 9};
+	}
+
+	@Test
+	void testArray() {
+		int[] z = { 7, 3, 5, 9, 2 };
+		int[] q = { 2, 3, 5, 7, 9 };
 		sortIntArray(z);
 		assertArrayEquals(q, z);
-		
-		int[] m = {7, 3, 9, 5, 2};
+	}
+
+	@Test
+	void testMiddle() {
+		int[] m = { 7, 3, 9, 5, 2 };
 		assertEquals(5, findMiddle(m));
 	}
-	
-	//1. Complete the method so that the first two elements of the array are swapped
+
+	// 1. Complete the method so that the first two elements of the array are
+	// swapped
 	public static void swapArrayOfTwo(int[] arr) {
-		
+		if (arr[0] < arr[1]) {
+			int value = arr[0];
+			arr[0] = arr[1];
+			arr[1] = value;
+		}
 	}
-	
-	//2. Complete the method so that it sorts the array using a bubble sort.
-	//   Iterate through the array and when you find two elements that are out
-	//   of order, swap them. Repeat this until the array is in order.
+
+	// 2. Complete the method so that it sorts the array using a bubble sort.
+	// Iterate through the array and when you find two elements that are out
+	// of order, swap them. Repeat this until the array is in order.
 	public static void sortIntArray(int[] arr) {
-		
+		for (int i = 0; i < arr.length; i++) {
+			for (int f = 0; f < arr.length - 1; f++) {
+				if (arr[f] > arr[f + 1]) {
+					int value = arr[f];
+					arr[f] = arr[f + 1];
+					arr[f + 1] = value;
+				}
+			}
+		}
 	}
-	
-	//3. Complete the method so that it finds the middle number in the array.
-	//   *Hint* it helps to sort it first.
-	//   *Double Hint* Use the method you already wrote in step 2 to sort it
+
+	// 3. Complete the method so that it finds the middle number in the array.
+	// *Hint* it helps to sort it first.
+	// *Double Hint* Use the method you already wrote in step 2 to sort it
 	public static int findMiddle(int[] arr) {
-		return 0;
+		for (int i = 0; i < arr.length; i++) {
+			for (int f = 0; f < arr.length - 1; f++) {
+				if (arr[f] > arr[f + 1]) {
```

Comment:
This code looks like it was copy/pasted from the `sortIntArray` method.  Can you think of a DRYer way writing this?